lib/assessment.js

```diff
@@ -4,24 +4,31 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(arg) {
+  min = math.min();
 };
+//I used mdn
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+  return math.sum();
 };
```

Comment:
See above.

Here, you try to return the *invoked* method, which means it returns some other value. `math` is not `Math`, and `Math.sum` doesn't exist anyway.